.drone.yml

```diff
@@ -23,6 +23,14 @@ services:
     ports:
       - 6379
 
+  - name: redis6-2
```

Comment:
I think you can just change the current `redis6` redis instance to use redis 6.2, no need for a separate test instance.